storybook/stories/bar-stack/horizontal.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import StackedBartChart from 'react-native-svg-charts'
```

Comment:
import is wrong. It's `StackedBarChart` (without the 't') and it's a named export, not default, i.e `import { StackedBarChart } from 'react-native-svg-charts'`. Please test your code before pushing (this does not run)